Unix/tools/allocdump

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+#
+# Script to run to collect memory allocation information for all running OMI processes
+# Note that in order for this to work:
+#   1.  Omi processes must have been built with debug turned on
```

Comment:
Hmm. We should create an issue so that, when doing a debug build, a marker file is left around. Or perhaps set up an OMI version file (installinfo or something) that gives the OMI build version and if it's debug or not. Then we can verify that this is the case.

I'd like this script to be as robust as possible so customers know what to do if something goes wrong.